.github/workflows/binaries.yml

```diff
@@ -36,6 +36,9 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - run: rustup update
+    - name: install libudev-dev
+      run: |
+        sudo apt install -y libudev-dev
```

Comment:
You need to match the arch, like we're currently doing a [few lines below](https://github.com/stellar/stellar-cli/pull/1353/files#diff-bdfe89ee8491e51e3f81ada685c51d24188416d8b529b443ee6085a0e0fd4a4aL40).